app/backend/src/couchers/servicers/search.py

```diff
@@ -461,6 +461,19 @@ def UserSearch(self, request, context):
                         (1000 + request.search_in_area.radius) / 111111,
                     )
                 )
+            if request.HasField("search_in_viewport"):
```

Comment:
I will implement this check in the frontend, but maybe it´s worth implementing it in the backend as well

Reply:
It should be fine to send the backend anything, I think. Not doing it for any viewport in the frontend is a more rate-limiting thing.

Reply:
true... I said because I remember some conversations we had, when I have shown you the example of when a user search in Thailand, only a few results were sent, and I remember something like "we don't want to allow the user to search in big areas..." (if I remember correctly...)

Anyhow, I will implement the bbox limit in the frontend ;)

Reply:
I think that was the reason (maybe I'm wrong) why the backend only searchs 50km around an specific point

Reply:
The reason for not searching in big areas (countries) with the current system is that we currently only search around a point: if you put in "Australia" the pin ends up in the outback and no one is there. It needs to be a smaller locality so that the 50km means something!

Reply:
So you don't think we should limit it for any searches? Then I'd remove my TODOs :D

Reply:
I think it's fine for now: we can think about limits later. Right now it should not be a problem.